pkg/oc/cli/newapp/newapp.go

```diff
@@ -450,9 +450,14 @@ func (o *AppOptions) RunNewApp() error {
 	for _, item := range result.List.Items {
 		// these are all unstructured
 		unstructuredObj := item.(*unstructured.Unstructured)
+
 		obj, err := legacyscheme.Scheme.New(unstructuredObj.GroupVersionKind())
 		if err != nil {
-			return err
+			// if we are dealing with a type that is not registered in the scheme,
+			// simply log the error and return, as we would not have a case
+			// for specifically handling that type in the switch below anyway.
+			glog.V(4).Infof("Failed to find a registered API object for %s: %v", unstructuredObj.GroupVersionKind(), err)
+			return nil
```

Comment:
even if this is right, returning seems wrong since you would still want to process the rest of the list of Items.